community-operators/kubefed-operator/0.1.0/operator_v1alpha1_kubefedwebhook_crd.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: kubefedwebhooks.operator.kubefed.io
+spec:
+  group: operator.kubefed.io
+  names:
+    kind: KubeFedWebHook
+    listKind: KubeFedWebHookList
+    plural: kubefedwebhooks
+    singular: kubefedwebhook
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          anyOf:
+          - type: "nullable"
+          - type: object
+        status:
```

Comment:
This needs a version property. Look at my changes in the operator repo: https://github.com/openshift/kubefed-operator/pull/70/files#diff-74e9cd8b8a8ad1f4e69931290d33e0e8R35